index.html

```diff
@@ -1,14 +1,100 @@
+                        <figcaption>Hanging Lake, Aspen, CO</figcaption></figure>
+                    <figure>
+                        <img id="figure3"src="https://i.imgur.com/Oqn7Iw4.png" alt="blue water and what looks to be colored Rocky Mountains because of the light"/>  
+                        <figcaption>Alma, CO</figcaption></figure>
+            </div>
+            <!--to make the button clickable, I added the clickable class and a link to take it to -->
+            <div class="clickable"><a href="learn-more.html"><button id="learn">Tell Me More</button></a></div>
+        </div>
+        <!--Sadly I couldn't get text to go over my picture so I just left it as a background image-->
+        <div id="quote"><img src="https://i.imgur.com/asnRV1N.png" alt="the sky in Crested Butte, Colorado" class="background";/></div>
+
+        <div id="body">
+            <div id="aliens">We actually ended up having a pretty great summer...<br><br>
+                <img src="https://i.imgur.com/9acPuzi.png" alt="view of the Rockies from Alma, Colorado" class="sky";/><br><br></div>
+            <section id="continue">Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus reiciendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda omnis qui nemo saepe excepturi vero odit, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates expedita reiciendis veniam eos temporibus perspiciatis doloremque quae! Similique aspernatur laborum nam quos. Iste quis modi quos est hic voluptatem libero id, aut in dolorem iure, officia enim harum unde quia repudiandae dolore quo....<br>
+            <a href="continue-reading.html"><button id="reading">Continue Reading</button></a><hr><hr> </section>
+            <section id="awesome">I took these photos in this area!<br><br><br>
+            <img src="https://i.imgur.com/icyPfkv.png" alt="photo of a mountain goat standing on a rock formation in Vail, CO" class="goat";/>
+                    </section>
+            <section id="text">Lorem ipsum dolor sit amet consectetur adipisicing elit. Maxime, numquam commodi! Accusamus reiciendis distinctio quibusdam ipsa perferendis? Numquam perspiciatis pariatur quia ducimus impedit nobis, deleniti dignissimos ipsam assumenda omnis qui nemo saepe excepturi vero odit, similique voluptate. Sit, recusandae a assumenda unde sunt voluptates expedita reiciendis veniam eos temporibus perspiciatis doloremque quae! Similique aspernatur laborum nam quos. Iste quis modi quos est hic voluptatem libero id, aut in dolorem iure, officia enim harum unde quia repudiandae dolore quo....<br>
+                      <a href="continue-reading.html"><button id="reading">Continue Reading</button>  </section></a>
+        </div>
+<!--My photos wouldn't upload to VS Code, so I used imgur to upload all my own photos and use those urls to share my photos here-->

```

Comment:
If you want to chat about loading images through VSCode we can take a little bit of time to do that. It's not a terribly hard thing to do but can be a little confusing when first trying it